OpenRA.Mods.Common/Traits/CashTrickler.cs

```diff
@@ -27,6 +27,9 @@ public class CashTricklerInfo : PausableConditionalTraitInfo, IRulesetLoaded
 		[Desc("Amount of money to give each time.")]
 		public readonly int Amount = 15;
 
+		[Desc("If false, the cash ticks will show up, but not actually give the cash. Used for fakes.")]
+		public readonly bool TransfersCash = true;
```

Comment:
IMO this naming is still problematic: If I didn't read the description I would assume that `TransfersCash: False` implies that it should be transferring resources instead...

I do quite strongly feel that this behaviour would be better represented with a separate `FakeCashTricker` trait.

Reply:
Can we skip this entirely? We don't have a proper test case in this repository and you can achieve the same by creating a `FakeCashTrickler` trait.